FASTQ files described as analysis_file instead of sequence_file

… for the following request:
```
http 'https://service.azul.data.humancellatlas.org/fetch/manifest/files?filters={"genusSpecies":{"is":["Homo sapiens"]},"fileFormat":{"is":[".bed.gz",".csv",".h5",".h5ad",".mtx",".mtx.gz",".rds",".tsv",".tsv.gz",".txt",".txt.gz","asc","bai","bam","bed.gz","cloupe","csv","csv.gz","fastq","fastq.gz","fq","fq.gz","gz","h5","h5.gz","h5ad","h5ad.gz","h5ad.tar.gz","h5ad.zip","jpg","jpg.gz","json","json.gz","loom","loom.gz","md5","mtx","mtx.gz","nd2","ndpi","pkc.gz","png","png.gz","RAW.tar","RData","RData.gz","Rdata.gz","rds","RDS","Rds","rds.gz","Rds.gz","RDS.gz","Robj","Robj.gz","son.gz","tab","tar","tar.gz","tif","tif.gz","tiff","tsv","tsv.gz","txt","txt.gz","xls.gz","xlsx","zip","zip.gz"]}}&catalog=dcp31&format=terra.bdbag'
HTTP/1.1 200 OK
Access-Control-Allow-Headers: Authorization,Content-Type,X-Amz-Date,X-Amz-Security-Token,X-Api-Key
Access-Control-Allow-Origin: *
Connection: keep-alive
Content-Length: 697
Content-Type: application/json
Date: Tue, 10 Oct 2023 16:19:25 GMT
Strict-Transport-Security: max-age=31536000; includeSubDomains
Via: 1.1 538a08eba98551a196e344df4d0dda06.cloudfront.net (CloudFront)
X-Amz-Cf-Id: N3RPPgGlWt8axcrqB3JjR3rryDC8O0SfVIqO0SgsI9AmXG2Z3fSS8A==
X-Amz-Cf-Pop: LAX50-C1
X-Amzn-Trace-Id: Root=1-6525797e-2aade9383dd1abec0183182a;Sampled=0;lineage=2808a99a:0
X-Cache: Miss from cloudfront
x-amz-apigw-id: Ml_r2EoGoAMFkTg=
x-amzn-RequestId: 660cb271-b58a-4339-aa63-5d7c46204dc1

{
    "CommandLine": {
        "bash": "curl 'https://service.azul.data.humancellatlas.org/fetch/manifest/files?token=eyJleGVjdXRpb25faWQiOiAiNjQ4MGU4NGItYzJlYS00YTQyLWJlOWUtMzcyMzY4ZmY5Nzc1IiwgInJlcXVlc3RfaW5kZXgiOiAwLCAid2FpdF90aW1lIjogMX0%3D'",
        "cmd.exe": "curl.exe \"https://service.azul.data.humancellatlas.org/fetch/manifest/files?token=eyJleGVjdXRpb25faWQiOiAiNjQ4MGU4NGItYzJlYS00YTQyLWJlOWUtMzcyMzY4ZmY5Nzc1IiwgInJlcXVlc3RfaW5kZXgiOiAwLCAid2FpdF90aW1lIjogMX0%3D\""
    },
    "Location": "https://service.azul.data.humancellatlas.org/fetch/manifest/files?token=eyJleGVjdXRpb25faWQiOiAiNjQ4MGU4NGItYzJlYS00YTQyLWJlOWUtMzcyMzY4ZmY5Nzc1IiwgInJlcXVlc3RfaW5kZXgiOiAwLCAid2FpdF90aW1lIjogMX0%3D",
    "Retry-After": 1,
    "Status": 301
}
…
http 'https://service.azul.data.humancellatlas.org/fetch/manifest/files?token=eyJleGVjdXRpb25faWQiOiAiNjQ4MGU4NGItYzJlYS00YTQyLWJlOWUtMzcyMzY4ZmY5Nzc1IiwgInJlcXVlc3RfaW5kZXgiOiAwLCAid2FpdF90aW1lIjogMX0%3D'
HTTP/1.1 500 Internal Server Error
Access-Control-Allow-Headers: Authorization,Content-Type,X-Amz-Date,X-Amz-Security-Token,X-Api-Key
Access-Control-Allow-Origin: *
Connection: keep-alive
Content-Length: 782
Content-Type: text/plain
Date: Tue, 10 Oct 2023 16:22:52 GMT
Strict-Transport-Security: max-age=31536000; includeSubDomains
Via: 1.1 ff59c1cd74c841ab9a3ebd5370e3b24a.cloudfront.net (CloudFront)
X-Amz-Cf-Id: KleT1ulXtmFQdiMMUNi2SGUqti6VYB6rM-pcxioTdKr86kAClxBvoA==
X-Amz-Cf-Pop: LAX50-C1
X-Amzn-Trace-Id: Root=1-65257a5c-701b2d5b4ed8a9955963012c;Sampled=0;lineage=2808a99a:0
X-Cache: Error from cloudfront
x-amz-apigw-id: MmAOgHCfIAMF6FA=
x-amzn-RequestId: 83945c47-be64-4b55-8385-c19b998f0980

Traceback (most recent call last):
  File "/var/task/chalice/app.py", line 1913, in _get_view_function_response
    response = view_function(**function_args)
  File "/var/task/app.py", line 1306, in fetch_file_manifest
    return _file_manifest(fetch=True)
  File "/var/task/app.py", line 1328, in _file_manifest
    return app.manifest_controller.get_manifest_async(self_url=app.self_url,
  File "/var/task/azul/service/manifest_controller.py", line 148, in get_manifest_async
    token_or_state = self.async_service.inspect_generation(token)
  File "/var/task/azul/service/async_manifest_service.py", line 100, in inspect_generation
    raise StateMachineError(status, output)
azul.service.async_manifest_service.StateMachineError: ('Failed to generate manifest', 'FAILED', None)
```

This caused an alarm notification. CW logs indicate that this is due to a KeyError.
```
[ERROR] KeyError: 'read_index'
Traceback (most recent call last):
  File "/var/task/azul/chalice.py", line 166, in patched_event_source_handler
    return old_handler(self_, event, context)
  File "/var/task/chalice/app.py", line 1752, in __call__
    return self.handler(event_obj)
  File "/var/task/chalice/app.py", line 1708, in __call__
    return self.handler(request, self.next_handler)
  File "/var/task/azul/chalice.py", line 191, in _lambda_context_middleware
    return get_response(event)
  File "/var/task/chalice/app.py", line 1698, in __call__
    return self._original_func(event.to_dict(), event.context)
  File "/var/task/app.py", line 1339, in generate_manifest
    return app.manifest_controller.get_manifest(event)
  File "/var/task/azul/service/manifest_controller.py", line 80, in get_manifest
    result = self.service.get_manifest(format_=ManifestFormat(state['format_']),
  File "/var/task/azul/service/manifest_service.py", line 378, in get_manifest
    partition = generator.write(object_key, partition)
  File "/var/task/azul/service/manifest_service.py", line 1068, in write
    file_path, base_name = self.create_file()
  File "/var/task/azul/service/manifest_service.py", line 1523, in create_file
    self._samples_tsv(samples_tsv)
  File "/var/task/azul/service/manifest_service.py", line 1616, in _samples_tsv
    qualifier = f"fastq_{file['read_index']}"
```

Original StepF exec: [azul-manifest-prod:3dcb2e4a-c485-4476-be4b-5fe3839e7a97](https://us-east-1.console.aws.amazon.com/states/home?region=us-east-1#/v2/executions/details/arn:aws:states:us-east-1:542754589326:execution:azul-manifest-prod:3dcb2e4a-c485-4476-be4b-5fe3839e7a97)
Repro StepF exec: [azul-manifest-prod:6480e84b-c2ea-4a42-be9e-372368ff9775](https://us-east-1.console.aws.amazon.com/states/home?region=us-east-1#/v2/executions/details/arn:aws:states:us-east-1:542754589326:execution:azul-manifest-prod:6480e84b-c2ea-4a42-be9e-372368ff9775)
<!--
Add your description here. For suspected bugs, the steps required to reproduce
the problem, as well as the expected and actual outcome should be included in
the description. Solutions should be proposed in a  separate comment, not the
issue description. Please do not modify or delete the checklist below. The
checklist should remain at the end of the issue description.
-->

- [ ] Security design review completed; the Resolution of this issue does **not** …
  - [ ] … affect authentication; for example:
    - OAuth 2.0 with the application (API or Swagger UI)
    - Authentication of developers with Google Cloud APIs
    - Authentication of developers with AWS APIs
    - Authentication with a GitLab instance in the system
    - Password and 2FA authentication with GitHub
    - API access token authentication with GitHub
    - Authentication with 
  - [ ] … affect the permissions of internal users like access to
    - Cloud resources on AWS and GCP
    - GitLab repositories, projects and groups, administration
    - an EC2 instance via SSH
    - GitHub issues, pull requests, commits, commit statuses, wikis, repositories, organizations
  - [ ] … affect the permissions of external users like access to
    - TDR snapshots
  - [ ] … affect permissions of service or bot accounts
    - Cloud resources on AWS and GCP
  - [ ] … affect audit logging in the system, like
    - adding, removing or changing a log message that represents an auditable event
    - changing the routing of log messages through the system
  - [ ] … affect monitoring of the system
  - [ ] … introduce a new software dependency like
    - Python packages on PYPI
    - Command-line utilities
    - Docker images
    - Terraform providers
  - [ ] … add an interface that exposes sensitive or confidential data at the security boundary
  - [ ] … affect the encryption of data at rest
  - [ ] … require persistence of sensitive or confidential data that might require encryption at rest
  - [ ] … require unencrypted transmission of data within the security boundary
  - [ ] … affect the network security layer; for example by 
    - modifying, adding or removing firewall rules
    - modifying, adding or removing security groups
    - changing or adding a port a service, proxy or load balancer listens on
- [ ] Documentation on any unchecked boxes is provided in comments below

--- a/deployments/prod/environment.py
+++ b/deployments/prod/environment.py
@@ -910,7 +910,7 @@ dcp31_sources = mkdict(dcp30_sources, 399, mkdelta([
     mksrc('datarepo-2abbf49d', 'hca_prod_e456c042f6b64ceca3381a8ef80bd779__20230905_dcp2_20230905_dcp31', 291),
 ]))
 
-dcp32_sources = mkdict(dcp31_sources, 407, mkdelta([
+dcp32_sources = mkdict(dcp31_sources, 406, mkdelta([
     mksrc('datarepo-6885133e', 'hca_prod_0792db3480474e62802c9177c9cd8e28__20220107_dcp2_20231002_dcp32', 1451),
     mksrc('datarepo-03b554f8', 'hca_prod_0911cc0406d64ffc8318b90b0039e8ad__20230905_dcp2_20231002_dcp32', 42),
     mksrc('datarepo-a5249352', 'hca_prod_279f176633194e3c9f996fb59ba9b3e5__20230815_dcp2_20231002_dcp32', 3),
@@ -930,6 +930,7 @@ dcp32_sources = mkdict(dcp31_sources, 407, mkdelta([
     mksrc('datarepo-d13e36e7', 'hca_prod_53c53cd481274e12bc7f8fe1610a715c__20220117_dcp2_20231002_dcp32', 35),
     mksrc('datarepo-0287a0ba', 'hca_prod_577c946d6de54b55a854cd3fde40bff2__20220117_dcp2_20231002_dcp32', 21),
     mksrc('datarepo-4d2eebce', 'hca_prod_5a54c6170eed486e8c1a8a8041fc1729__20230616_dcp2_20231002_dcp32', 53),
+    mksrc('datarepo-2917ceb6', 'hca_prod_5b3285614a9740acb7ad6a90fc59d374__20220117_dcp2_20230314_dcp25', 409, pop),
     mksrc('datarepo-069ac8d2', 'hca_prod_67a3de0945b949c3a068ff4665daa50e__20220117_dcp2_20231002_dcp32', 735),
     mksrc('datarepo-7be1db07', 'hca_prod_6936da41369246bbbca1cd0f507991e9__20230905_dcp2_20231003_dcp32', 1081),
     mksrc('datarepo-b59acd40', 'hca_prod_72ff481856924bbc8886e47763531023__20230929_dcp2_20231002_dcp32', 32),

